iam/sc-s3-launchrole.yaml

```diff
@@ -6,6 +6,7 @@ Resources:
       RoleName: SCS3LaunchRole
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/AmazonS3FullAccess
+        - arn:aws:iam::aws:policy/AWSLambdaFullAccess

```

Comment:
do you want to give the launch role full access to all lambdas rather than targeted access?

Reply:
good question.  I think with the way that the SC is designed, in that just about any customization requires a lambda, I was thinking it might be general policy to give every SC role AWSLambdaFullAccess.  Or we can narrow down now and expand lambda access as we go? What do you think?

Reply:
I think we could try for more targeted access, if we can get that to work, as in [cfn-ssm-param-policy](https://github.com/Sage-Bionetworks/scipoolprod-infra/blob/master/templates/cfn-ssm-param-policy.yaml).

Reply:
done